packages/system/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.26.1"
```

```suggestion
- version: "1.25.4"
```

We don't need to bump the minor version for a description change. The `1.26.0-next` below is not actually released yet so we're still on 1.25.x.